cura/Settings/ContainerManager.py

```diff
@@ -522,7 +333,7 @@ def updateQualityChanges(self):
 
             self._performMerge(quality_changes, stack.getTop())
```

Comment:
Use _stack.userChanges_ instead of  _stack.getTop()_

Reply:
Changed in ccd4bb285

Reply:
This commit is reverted in f304e7f because those PerObjectContainerStacks are not CuraContainerStacks, so we cannot use properties such as userChanges.